### ISSUE TITLE
Release

### ISSUE THREAD

**Issue by asmeurer** [drdoctr/doctr#167]
We need to get a release out. Which existing pull requests should be merged first? Do we need to do any more testing?

I'm also taking volunteers to [do the release](https://drdoctr.github.io/doctr/releasing.html) (though happy to do it myself again if no one wants to).

**Comment by gforsyth** [comment]
I don't think we need #163 before a release since I don't know what I want that to do. I'd like to have #164 in, but Travis is still weirdly stalled on that and it needs a few changes

**Comment by gforsyth** [comment]
And I'm happy to do the release.

**Comment by asmeurer** [comment]
Let's just finish #164 in a new branch.

**Comment by gforsyth** [comment]
Pushed up to `deploy_to_any`
